adaptive_components/pubspec.yaml

```diff
@@ -4,13 +4,17 @@ version: 0.0.1
 homepage: https://github.com/material-components/material-components-flutter-adaptive
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  adaptive_breakpoints: ^0.0.3
-  adaptive_navigation: ^0.0.1
+  adaptive_breakpoints:
+    # TODO: Return to normal.
```

Comment:
remove TODOs

Reply:
As of now, since NNBD is not released as beta yet, we cannot publish new packages to pub.dev. Once NNBD beta is released, we will be able to publish them in order and change to pub dependencies.
I updated the TODO.